tools/terraform/ecs.tf

```diff
@@ -113,6 +106,12 @@ resource "aws_cloudwatch_log_group" "backend" {
   retention_in_days = 1
 }
 
+data "aws_region" "current" {}
+
+data "aws_secretsmanager_secret" "license" {
```

Comment:
I don't think we want to add this back in, not everyone running this TF has a license, or rather even this secret provisioned.